api/controllers/HealthController.js

```diff
@@ -0,0 +1,22 @@
+const { version: pkgVersion } = require('../../package.json');
+
+const isDatabaseAvailable = async () => {
+  return await sails.helpers.getApiHealth();
+};
```

Comment:
I'm open to keeping this, but penny for your thoughts--why create the `isDatabaseAvailable` function rather than calling `await sails.helpers.getApiHealth();` directly in the endpoint?

Reply:
It's a one liner so we can definitely merge it and remove the additional function. What I originally thought is, the `isDatabaseAvailable` function in this controller will hold the logic to get that value. However that's not true as before I was delegating that to a hook and now to a helper (one liners). I'll make the change :zap: